core/src/main/java/org/apache/gravitino/catalog/TableOperationDispatcher.java

```diff
@@ -348,34 +364,45 @@ private void importTable(NameIdentifier identifier) {
       // of external system to correct it.
       LOG.warn(
           "The Table uid {} existed but still need to be imported, this could be happened "
-              + "when Table is renamed by external systems not controlled by Gravitino. In this case, "
-              + "we need to overwrite the stored entity to keep the consistency.",
+              + "when Table is renamed by external systems not controlled by Gravitino. In this "
+              + "case, we need to overwrite the stored entity to keep the consistency.",
           stringId);
       uid = stringId.id();
     } else {
       // If entity doesn't exist, we import the entity from the external system.
       uid = idGenerator.nextId();
     }
 
+    AuditInfo audit =
+        AuditInfo.builder()
+            .withCreator(table.auditInfo().creator())
+            .withCreateTime(table.auditInfo().createTime())
+            .withLastModifier(table.auditInfo().lastModifier())
+            .withLastModifiedTime(table.auditInfo().lastModifiedTime())
+            .build();
+    List<ColumnEntity> columnEntityList =
+        toColumnEntities(table.tableFromCatalog().columns(), audit);
```

Comment:
The final modifier of the table could potentially be the column creator, however, it appears that we may not have the capability to address this scenario, right?